packages/app/pages/api/ethpass.ts

```diff
@@ -28,10 +28,18 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
             tokenId,
             signature,
             signatureMessage,
-            platform: "apple",
-            pass: {
+            platform,
+            templateId: process.env.ETHPASS_TEMPLATE_ID,
+            barcode: { message: "Thanks for participating at Cabin’s first inaugural GUILD GAMES!"},
+            pass: {   
               description,
+              headerFields: [{
+                key: "header1",
+                value: "DEMO PASS",
+                textAlignment: "PKTextAlignmentNatural"
+              }],
```

Comment:
what does this do?

Reply:
![IMG_163F5536D22E-1](https://user-images.githubusercontent.com/3741055/160216690-c7d78e59-4a3f-47d7-826c-0b9abc4deb08.jpeg)

Reply:
It's an empty canvas right now so we can decide what other info we want to put here. I was thinking we can pull the name and symbol from the contract as a start.

I didn't get around to customizing the Google pass quite yet so it's gonna look like the one from our site. I'll work on it starting next week and ping all of you for some feedback.